packages/SwingSet/test/test-transcriptlessness.js

```diff
@@ -0,0 +1,57 @@
+/* global require */
+// eslint-disable-next-line import/order
+import { test } from '../tools/prepare-test-env-ava';
+
+import { provideHostStorage } from '../src/hostStorage';
+import { buildVatController } from '../src/index';
+
+function capdata(body, slots = []) {
```

Comment:
not much of a win, but `test/util.js` has a `capargs` that you can import instead of redefining it here